dev/contr.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+All contributions must be made via a Pull Request to the main GitHub-Repo.
+
+For a pull request to get merged, the following conditions must be fulfilled:
+
+- all applicable pipelines must succeed
+- at least two approvals must be made for the patch to be merged after 24 hours, or at least four approvals must be made to merge the patch immediately
+- the PR must neither be controversial or blocked (indicated by tags)
+
+**Fundamental changes, like changes in the database scheme or other disputed areas, require at least four reviews. Those changes should be discussed in a developer meeting. A "Fundamental"-Tag should indicate those changes.**
+
+The Documentation may be also expanded using pull requests. For the documentation repository, two approvals are enough for immediate merge.
+
+Usually, commits from a single PR are squashed and merged.
+
+PRs need to have a meaningful name, and, if applicable, a description.
```

Comment:
I would condense this into a listing like so:

```suggestion
The Documentation may be also expanded using pull requests (PRs). Pull requests need need to follow a few guidelines:

- A PR may be merged immediately with a minimum of only two approvals.
- Multiple commits must be stashed before or during merging, exceptions confirm the rule.
- PRs require a meaningful title and, if needed to explain the changes or the rational behind them, a proper description.
```

Reply:
We could also, I would even prefer that, use the same set of default guidelines (e.g. Title, squashing, approvals, guidelines for fundamental or controversial changes) for the Code and the Documentation and only defining differences, for example that a Documentation PR may be merged immediately when two contributors approved of it.

Reply:
Point two: Really a git stash? Not squash?
The other points are looking good, thanks!

Reply:
I keep messing those two up... Fixed